services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/TopicCleanupService.java

```diff
@@ -74,13 +82,17 @@ public class TopicCleanupService extends AbstractVeniceService {
   private boolean isRTTopicDeletionBlocked = false;
   private boolean isLeaderControllerOfControllerCluster = false;
   private long refreshQueueCycle = Time.MS_PER_MINUTE;
+  private Optional<PubSubAdminAdapter> apacheKafkaAdminAdapter;
```

Comment:
Is there any benefit of using `Optional<>`?  If there's not much value, we can use `null` if it's absent.